source/extensions/extensions_metadata.yaml

```diff
@@ -539,6 +539,16 @@ envoy.quic.proof_source.filter_chain:
   - envoy.quic.proof_source
   security_posture: robust_to_untrusted_downstream
   status: alpha
+envoy.udp_packet_writer.default:
+  categories:
+  - envoy.udp_packet_writer
+  security_posture: robust_to_untrusted_downstream_and_upstream
+  status: alpha
+envoy.udp_packet_writer.gso:
+  categories:
+  - envoy.udp_packet_writer
+  security_posture: robust_to_untrusted_downstream_and_upstream
+  status: alpha
```

Comment:
I would probably mark these as non-alpha as people have been implicitly using them for a long time?

Reply:
Sure, done. I wondered if it might make sense to mark them as alpha until we've actually used the protos, but fair enough.